examples/server/server.cpp

```diff
@@ -453,23 +487,33 @@ int main(int argc, char ** argv) {
         std::vector<float> pcmf32;               // mono-channel F32 PCM
         std::vector<std::vector<float>> pcmf32s; // stereo-channel F32 PCM
 
-        // write to temporary file
-        const std::string temp_filename = "whisper_server_temp_file.wav";
-        std::ofstream temp_file{temp_filename, std::ios::binary};
+        // write file to temporary file
+        std::ofstream temp_file{filename, std::ios::binary};
         temp_file << audio_file.content;
         temp_file.close();
 
+        // if file is not wav, convert to wav
+        if(sparams.ffmpeg_converter) {
+            std::string converted_filename = "converted_" + filename;
+            // ffmpeg command
+            std::string ffmpeg_cmd = "ffmpeg -i " + filename + " -ar 16000 -ac 1 -c:a pcm_s16le " + converted_filename + " 2>&1"; // Redirect stderr to stdout
+
+            executeConvertToWAV(ffmpeg_cmd);
```

Comment:
since we ignore the string on the return here, perhaps make it into void?

Reply:
Also, if the conversion failed we need to let the user know somehow. You could do like i did after "::read_wav" where i send a error_resp via "res.set_content(error_resp)". The error could be something like:

{"error": "File conversion failed."}
or and also 
{"error": "File could not be converted due to no ffmpeg installed."}